javav2/example_code/route53recoverycluster/src/main/resources/endpoints.json

```diff
@@ -0,0 +1,24 @@
+{
+  "Endpoints": [
+    {
+      "Endpoint": "https://e3c5ab77.route53-recovery-cluster.us-west-2.amazonaws.com/v1",
+      "Region": "us-west-2"
+    },
+    {
+      "Endpoint": "https://da6935e3.route53-recovery-cluster.ap-southeast-2.amazonaws.com/v1",
+      "Region": "ap-southeast-2"
+    },
+    {
+      "Endpoint": "https://329ac8cc.route53-recovery-cluster.us-east-1.amazonaws.com/v1",
+      "Region": "us-east-1"
+    },
+    {
+      "Endpoint": "https://18c250a2.route53-recovery-cluster.ap-northeast-1.amazonaws.com/v1",
+      "Region": "ap-northeast-1"
+    },
+    {
+      "Endpoint": "https://af73a750.route53-recovery-cluster.eu-west-1.amazonaws.com/v1",
+      "Region": "eu-west-1"
+    }
+  ]
+}
```

Comment:
I think these endpoints are specific to a cluster, so they shouldn't be included as part of the example, or should use example numbers to make it clear they are for example only.

Reply:
Updated to example endpoints